app/src/main/res/values/styles.xml

```diff
@@ -3,7 +3,7 @@
 
     <style name="TextAppearance">
         <item name="android:fontFamily">@font/bai_jamjuree_regular</item>
-        <item name="android:textColor">@color/pastel_purple</item>
+        <item name="android:textColor">@color/white</item>
```

Comment:
Create separate color resources to be called outside colors.xml
This way if you need to implement dark mode, you can have a colors file for dark mode with the exact same names for colors, and only the specific colors like `white` `purple` might be different

For example have a color res `text_color` that references white and in the dark mode colors file you'll have the same `text_color` res name but it might reference red. This way you only have to change the dark mode colors instead of everywhere in your code.

I suggest applying this change wherever necessary 😁

Reply:
On a real life project this saves a lot of development time if the client decides they want dark mode after the initial release